ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -87,8 +152,13 @@ class MainService : Service() {
     @Inject
     lateinit var modelsRepository: ModelRepositoryImpl
 
+    @Inject
+    lateinit var offloadingServiceRepositoryImpl: OffloadingServiceRepositoryImpl
+
     private var initialized = false
     private var serverInfoMap = mutableMapOf<String, ServerInfo>()
+    private var serviceMap = mutableMapOf<Int, OffloadingServiceStatus>()
+    private var gServiceId = 1
```

Comment:
If you want to use a persistent variable that is initialized only once when the app is installed, it should be moved to the room or datastore. I think this variable could be initialized every time the service restarted.

Reply:
I am considering using the `PrimaryKey(autoGenerate = true)` option.